helm/azure-operator-chart/templates/deployment.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
```

Comment:
What version of k8s is the apps/v1 started being supported? Are we sure all azure installations are at least that version?

Reply:
It was added in 1.9 so we shouldn't have problems. We need to switch to `apps/v1` as `extensions/v1beta1` is removed in 1.16. 

https://github.com/kubernetes/kubernetes/issues/61430

Reply:
It started on `1.9`, and it's required for `1.16`.

Reply:
apps was GA since 1.9, it's safe to switch, latest we support is 1.13
https://kubernetes.io/blog/2017/12/kubernetes-19-workloads-expanded-ecosystem/

Reply:
`1.14` is the oldest version I could find in our installations.